### ISSUE TITLE
Requesting a sed icon

### ISSUE THREAD

**Issue by glennj** [exercism/sed#6]
AFAIK sed does not have a logo.

Perhaps we can request a very simple one: some text in black on a white background.

* `sed`
* `s///`

The request will be made at https://github.com/exercism/website-icons/issues

**Comment by jimmytty** [comment]
Yes, sed need a track logo. The name _sed_ is the best option because _s///_ is a common command for famous tools like _vim_, _vi_, _ed_.

**Comment by jimmytty** [comment]
DONE. 

[https://github.com/exercism/website-icons/issues/175#issue-2399142625](https://github.com/exercism/website-icons/issues/175#issue-2399142625)
